lib/db-sync-progress.js

```diff
@@ -40,6 +40,11 @@ module.exports = function (db, opts) {
       updateFeed(feed)
     }
     listeners.push({ feed, listener })
+    // XXX: This is important because at least one progress event should be
+    // emitted to ensure the state moves from 'started' to 'progress'
+    // indicating up the stack that progress tracking has begun
+    // for more accurate state
+    updateFeed()
```

Comment:
Actually I don't think this is doing anything but it would be nice if it did. Right now if you re-sync with a device that has missing data, it'll say 'started' for 20 seconds until the heartbeat destroys the connection, which is a bit confusing.

Reply:
I /guess/ that the progress listeners aren't yet attached during the 'started' state, is there a reason why we add them during the handshake accept and not during init?

Reply:
I don't think so? I think it would be fine to do that. This hack is ok with me in the short-term.

Reply:
Now this will work due to change below!